docs/develop/showcase.md

```diff
@@ -0,0 +1,96 @@
+---
+id: showcase
+title: Showcase
+---
+
+import ThemedImage from '@theme/ThemedImage';
+
+
+
+## Wallets
+
+Wallets are like the swiss army knife of the web3.
+Your wallet will store private information like your secret for your DID or your KILT credentials.
+Each time a website wants to know something about you, the wallet will ask for your permission to share or sign information.
+
+
+<a href="https://sporran.org">
+    <ThemedImage
+    width="200"
+    alt="Sporran Logo"
+    sources={{
+        light: '/img/showcase/sporran_light.svg',
+        dark: '/img/showcase/sporran_dark.svg',
+    }}
+    />
+</a>
+
+Sporran is the first wallet with KILT support.
+It is maintained by _BOTLabs Trusted Entity B.T.E GmbH_ and the [source code is open source](https://github.com/BTE-Trusted-Entity/sporran-extension).
+
+## Apps
```

Comment:
The  apps are visually more separated, than the whole wallet to the apps section. Could you  add some whitespace here?

Reply:
I also made the logos smaller so that the headline is more prominent than the apps.

would love to have the text floating next to the logo. But links aren't working and resizing the image doesn't work either. My CSS is not enough for this madness!

![Screenshot 2022-05-20 at 15 30 54](https://user-images.githubusercontent.com/14820950/169539514-51203073-ead9-4b86-a9b9-8f5f1dfb19f9.png)